routing.rst

```diff
@@ -1333,68 +1333,71 @@ Use the ``RedirectController`` to redirect to other routes and URLs:
             xsi:schemaLocation="http://symfony.com/schema/routing
                 https://symfony.com/schema/routing/routing-1.0.xsd">
 
-            <route id="doc_shortcut" path="/doc"
-                   controller="Symfony\Bundle\FrameworkBundle\Controller\RedirectController">
-                <default key="route">doc_page</default>
+            <redirect-route id="doc_shortcut" path="/doc"
+                    redirect-to-route="doc_page"
+                    <!-- redirections are temporary by default (code 302) but you can make them permanent (code 301)-->
+                    permanent="true">
                 <!-- optionally you can define some arguments passed to the route -->
                 <default key="page">index</default>
                 <default key="version">current</default>
-                <!-- redirections are temporary by default (code 302) but you can make them permanent (code 301)-->
-                <default key="permanent">true</default>
                 <!-- add this to keep the original query string parameters when redirecting -->
                 <default key="keepQueryParams">true</default>
                 <!-- add this to keep the HTTP method when redirecting. The redirect status changes:
                      * for temporary redirects, it uses the 307 status code instead of 302
                      * for permanent redirects, it uses the 308 status code instead of 301 -->
                 <default key="keepRequestMethod">true</default>
-            </route>
+            </redirect-route>
 
-            <route id="legacy_doc" path="/legacy/doc"
-                   controller="Symfony\Bundle\FrameworkBundle\Controller\RedirectController">
+            <url-redirect-route id="legacy_doc" path="/legacy/doc"
                 <!-- this value can be an absolute path or an absolute URL -->
-                <default key="path">https://legacy.example.com/doc</default>
+                redirect-to-url="https://legacy.example.com/doc"
                 <!-- redirections are temporary by default (code 302) but you can make them permanent (code 301)-->
-                <default key="permanent">true</default>
-            </route>
+                permanent="true">
+            </url-redirect-route>
         </routes>
 
     .. code-block:: php
 
         // config/routes.php
         use App\Controller\DefaultController;
-        use Symfony\Bundle\FrameworkBundle\Controller\RedirectController;
         use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
```

Comment:
Should it used the RoutingConfigurator of the bundle or the component ?

Reply:
The bundle, good catch! thx :)